charts/fluid-dataloader/jindocache/templates/cronjob.yaml

```diff
@@ -97,6 +97,8 @@ spec:
               {{- $pathReplicas = cat $pathReplicas ( default 1 .replicas ) ":"}}
               {{- end }}
               {{- $pathReplicas = $pathReplicas | nospace | trimSuffix ":"}}
+              resources:
```

Comment:
I think it's better to determine if .Values.dataloader.resources is nil?

Reply:
Done.